api/src/team/infrastructure/repositories/certification-center-membership.repository.js

```diff
@@ -63,6 +64,30 @@ const countActiveMembersForCertificationCenter = async function (certificationCe
   return count;
 };
 
+/**
+ * Disable active members of a certification center
+ *
+ * @param {Object} params
+ * @param {string} params.certificationCenterId
+ * @param {date} params.disabledAt
+ * @param {string} params.updatedByUserId
+ */
+
+const disableMembershipsByCertificationCenterId = async function ({
```

Comment:
Ne pas mettre de paraphrase inutile : 

```suggestion
/**
 * @param {Object} params
 * @param {string} params.certificationCenterId
 * @param {date} params.disabledAt
 * @param {string} params.updatedByUserId
 */

const disableMembershipsByCertificationCenterId = async function ({
```